client/src/components/Verejne/index.js

```diff
@@ -5,29 +5,54 @@ import './Verejne.css'
 import Legend from './Legend'
 import {Input, ListGroup, ListGroupItem, Badge} from 'reactstrap'
 import {connect} from 'react-redux'
-import {fetchEntities} from '../../actions/verejneActions'
+import {fetchEntities, selectEntity} from '../../actions/verejneActions'
 import {entitiesSelector} from '../../selectors'
 import {compose, lifecycle, branch, renderComponent, withProps, withState} from 'recompose'
 import Loading from '../Loading'
 import {sortBy, chunk, reverse} from 'lodash'
 import {VEREJNE_MAX_PAGE_ITEMS} from '../../constants'
 import Pagination from '../shared/GeneralPagination'
+import {isPolitician, hasContractsWithState} from './entityHelpers'
 
-import CircleIcon from 'react-icons/lib/fa/circle'
+import FilledCircleIcon from 'react-icons/lib/fa/circle'
+import CircleIcon from 'react-icons/lib/fa/circle-o'
+import MapIcon from './mapIcon.svg'
 
-const Verejne = ({entities, fetchEntities, pageCount, currentPage, setCurrentPage}) => (
+const renderListItemIcon = (entity) => {
+  if (entity.size > 1) {return <img src={MapIcon} style={{width: '2rem', height: '2rem'}} alt="listItemIcon" />}
+  const color = isPolitician(entity) ? '#e55600' : '#0062db'
```

Comment:
maybe do with classes (we're using classnames so it'll be nice), and we can then have these colors in variables